music21/harmony.py

```diff
@@ -2492,6 +2493,33 @@ def transpose(self: NCT, _value, *, inPlace=False) -> NCT | None:
 
 # ------------------------------------------------------------------------------
 
+class FiguredBassIndication(Harmony):
```

Comment:
I'm wondering what you think we should do about the existing `figure` property, which as of now will give:
```python
>>> fbi = harmony.FiguredBassIndication(root='D', figure='643')
>>> fbi
<FiguredBassIndication figures: >
>>> fbi.figure
'643'
```